fluidBuild.config.cjs

```diff
@@ -65,15 +65,10 @@ module.exports = {
 		"ts2esm": [],
 		"tsc": tscDependsOn,
 		"build:esnext": [...tscDependsOn, "^build:esnext"],
-		"build:test": [
-			// The tscDependsOn deps are not technically needed, but they are here because the fluid-build-tasks-tsc policy
-			// requires them. I don't want to change the policy right now.
-			...tscDependsOn,
-			"typetests:gen",
-			"tsc",
-			"api-extractor:commonjs",
-			"api-extractor:esnext",
-		],
+		// Generic build:test script should be replaced by :esm or :cjs specific versions.
+		// "tsc" would be nice to eliminate from here, but plenty of packages still focus
+		// on CommonJS.
+		"build:test": ["typetests:gen", "tsc", "api-extractor:commonjs", "api-extractor:esnext"],
```

Comment:
This seems like an odd combination. The test script name isn't specific, so if the package builds both commonjs and esm tests, shouldn't both be built by running build:test?

```suggestion
		"build:test": ["build:test:cjs", "build:test:esnext"],
```

Reply:
If package builds both esm and cjs tests, then those individual test build scripts should have their appropriate dependencies and this isn't needed. This config should be removed, but first the config needs moved to the places that still build tests from build:test. I was doing the minimal changes that are happy with current and new policy and avoiding looking at individual packages.

Reply:
> and this isn't needed
I'd say maybe we still want it? We have the single build/compile that covers commonjs and build:esnext, so having the same pattern for tests makes sense to me. That said, for another PR.